packages/i18n/src/translateWithMarkup.tsx

```diff
@@ -131,13 +136,18 @@ function TranslateWithMarkup({
 
   const node = new DOMParser().parseFromString(children, 'text/html').body
     .firstChild as Element;
-  return node
-    ? transform(
-        node,
-        mapping
-        //eslint-disable-next-line react/no-array-index-key -- Order should never change.
-      ).map((element, index) => <Fragment key={index}>{element}</Fragment>)
-    : [];
+  return node ? (
```

Comment:
I think it's more clear if we short-circuit the null return like:

```tsx
if (!node) {
  return null;
}
return ...
```

Reply:
(I know you didn't write this way in the first place, but just now that you're already here :P )